api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
@xiphiaz By having ```ValidationException``` extending ```HttpResponseException``` it actually already inherits the ```HttpExceptionInterface``` from the parent class, so it makes no practical difference to add the ```implements``` again. I'll add it to the code base though.

And I'll move the check to be inside the ```if ($e instanceof HttpExceptionInterface)```.